web3sTests/ENS/ENSTests.swift

```diff
@@ -126,5 +126,38 @@ class ENSTests: XCTestCase {
         
         waitForExpectations(timeout: 20)
     }
-    
+
+    func testGivenRopstenRegistry_TODO() {
```

Comment:
`testGiven3Addresses_whenOneDoesNotResolve_ReturnsCorrectValues` ?

Reply:
Fixed in https://github.com/argentlabs/web3.swift/pull/116/commits/8495db05bddf4235bce8c3e4a055d333937b1a87.